integration/walletmock/wallet.go

```diff
@@ -1,14 +1,38 @@
 package walletmock
 
 import (
+	"crypto/ecdsa"
+	"crypto/rand"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/google/uuid"
-	"github.com/obscuronet/obscuro-playground/go/common"
 )
 
 type Wallet struct {
 	Address common.Address
+	// TODO - Store key securely. Geth stores the key encrypted on disk.
+	Key keystore.Key
 }
 
 func New() Wallet {
-	return Wallet{Address: uuid.New().ID()}
+	id, err := uuid.NewRandom()
+	if err != nil {
+		panic(fmt.Sprintf("Could not create random uuid: %v", err))
+	}
+
+	privateKeyECDSA, err := ecdsa.GenerateKey(crypto.S256(), rand.Reader)
```

Comment:
I noticed there is a function: `crypto.GenerateKey()`, which makes exactly this call.
Shouldn't we use it?